staging/src/k8s.io/apiserver/pkg/endpoints/request/requestinfo.go

```diff
@@ -77,6 +77,11 @@ var specialVerbsNoSubresources = sets.NewString("proxy")
 // this list allows the parser to distinguish between a namespace subresource, and a namespaced resource
 var namespaceSubresources = sets.NewString("status", "finalize")
 
+// crSubresources contains allowed subresources of a CR:
+// https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#subresources
+// this list allows the parser to distinguish between a cluster-scoped CR's subresources and a list of namespace-scoped CRs
+var crSubresources = sets.NewString("status", "scale")
```

Comment:
I don't think this is unambiguous... wouldn't both of the following CRDs lead to an endpoint of `/apis/$group/$version/namespaces/foo/scale`?

* scope=namespaced, names.plural=scale
* scope=cluster, names.plural=namespaces, subresources.scale={}